portfolio/src/main/webapp/style.css

```diff
@@ -52,8 +64,63 @@ footer {
   background: black;
 }
 
+section {
+  animation: fadeIn 1s ease-in-out;
+}
+
+.section-content {
+  padding: 6rem;
+}
+
+.education-post {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 3rem 3rem;
+}
+
+.education-detail {
+  margin-left: 2rem;
+}
+
+#education,
+#work-experience,
+#projects,
+#skills,
+#interests {
+  display: none;
+}
+
+.project-row {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
```

Comment:
nit: Declarations are out of order here and some places below. https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Solved both, I forgot to check them before commiting